tests/bin/set-core-version.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
```

Comment:
It would be good to use the version in `node_modules` from `10up/cypress-wp-utils` if possible. If a bug is found in the script, it will be better to fix upstream. 

Of course, this may not be possible.

Reply:
Resolved in https://github.com/10up/maps-block-apple/pull/126/commits/66c017a735236a9757191b7b732cac5297ba5da6